simulator-samples/sample-swagger/src/test/java/org/citrusframework/simulator/SimulatorSwaggerIT.java

```diff
@@ -80,49 +80,94 @@ public void testUiInfo() {
     }
 
     @CitrusTest
-    public void testAddPet() {
+    public void addPetShouldSucceed() {
         variable("name", "hasso");
         variable("category", "dog");
         variable("tags", "huge");
         variable("status", "pending");
 
-        $(http().client(petstoreClient)
+        $(http().client(petstoreClientV2)
                 .send()
                 .post("/pet")
                 .message()
                 .accept(MediaType.APPLICATION_JSON_VALUE)
                 .contentType(MediaType.APPLICATION_JSON_VALUE)
                 .body(new Resources.ClasspathResource("templates/pet.json")));
 
-        $(http().client(petstoreClient)
+        $(http().client(petstoreClientV2)
                 .receive()
                 .response(HttpStatus.OK));
     }
 
     @CitrusTest
-    public void testDeletePet() {
+    public void addPetShouldFailOnMissingName() {
+        variable("category", "dog");
+        variable("tags", "huge");
+        variable("status", "pending");
+
+        $(http().client(petstoreClientV2)
+            .send()
+            .post("/pet")
+            .message()
+            .accept(MediaType.APPLICATION_JSON_VALUE)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .body(new Resources.ClasspathResource("templates/pet_invalid.json")));
+
+        $(http().client(petstoreClientV2)
+            .receive()
+            .response(HttpStatus.INTERNAL_SERVER_ERROR));
+    }
+
+    @CitrusTest
+    public void deletePetShouldSucceed() {
         variable("id", "citrus:randomNumber(10)");
 
-        $(http().client(petstoreClient)
+        $(http().client(petstoreClientV2)
                 .send()
                 .delete("/pet/${id}"));
 
-        $(http().client(petstoreClient)
+        $(http().client(petstoreClientV2)
                 .receive()
                 .response(HttpStatus.OK));
     }
 
     @CitrusTest
-    public void testGetPetById() {
+    public void deletePetShouldFailOnWrongIdFormat() {
+
+        $(http().client(petstoreClientV2)
+            .send()
+            .delete("/pet/xxxx"));
+
+        $(http().client(petstoreClientV2)
+            .receive()
+            .response(HttpStatus.INTERNAL_SERVER_ERROR));
+    }
+
+//    @CitrusTest
```

Comment:
is `DELETE` not supported, or..?

Reply:
See above, there is a delete test already.